src/core/ContentTile.tsx

```diff
@@ -81,14 +87,17 @@ const ContentTile: React.FC<ContentTileProps> = ({
           className={cn(
             "content-tile__feature relative p-3 h-[200px] pb-0 flex items-end justify-center overflow-hidden rounded-lg bg-neutral-100 dark:bg-neutral-1200 border border-neutral-300 dark:border-neutral-1000 transition-[border-color,height]",
             centerFeature && "items-center pb-3",
-
+            cta &&
+              !encapsulated &&
+              "group-hover/content-tile:border-neutral-500 dark:group-hover/content-tile:border-neutral-800 transition-colors",
             featureClassName,
           )}
         >
           <div
             className={cn(
               "flex justify-center max-h-[200px]",
-              !centerFeature && "pt-6 [&_img]:min-w-max [&_img]:h-[200px]",
+              !centerFeature && "[&_img]:min-w-max [&_img]:h-[200px]",
```

Comment:
okay, good that there is a define height so all they have to do is to provide a landscape image so it will be fully 100%